e2e/changelog_settings.yaml

```diff
@@ -0,0 +1,6 @@
+packages:
+  stevedore:
+    changelog:
+      "5.1.0":
+        - test 1
+        
```

Comment:
Relying on this file to retain trailing whitespace on this line is going to be brittle. My editor is configured to automatically strip whitespace. Please update the line that edits the file to include the needed whitespace to indent the new entry properly.

Reply:
I think it already does that: https://github.com/python-wheel-build/fromager/pull/325/files#diff-1cb0faa8083db060cd12ad0b5cbc65c23ecd1fc853f3bf7022cc8737d3c48ca5R53

https://github.com/python-wheel-build/fromager/pull/325/files#diff-4a06375158e17eabf9367b420b56d9e7f3e510c8d043938d6cf4b0f7c827ad2fR125